src/admin/content/views/ContentDetail.tsx

```diff
@@ -94,6 +94,14 @@ const ContentDetail: FunctionComponent<ContentDetailProps> = ({ history, match }
 										</th>
 										<td>{contentItem.content_type}</td>
 									</tr>
+									<tr>
+										<th>
+											<Trans i18nKey="admin/content/views/content-detail___beschermde-pagina">
+												Beschermde pagina:
+											</Trans>
+										</th>
+										<td>{contentItem.is_protected ? 'Ja' : 'Nee'}</td>
```

Comment:
hmm, het is wat veel om deze ook te vertalen zkr? Ze zullen dit wss nooit willen aanpassen als de copy. Maar mss als ze ooit een vertaling willen toevoegen dat het dan wel nuttig is? Dus mss voor de compleetheid toch een translation toevoegen?

Reply:
Had hier eerst een icon maar gwn ja en nee leek me duidelijker, ik zal het toevoegen.